script.module.kodiswift/addon.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.kodiswift" name="kodiswift" provider-name="Aaron Frase (Sinap); TJ Jiang" version="0.0.12+matrix.1">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+    <import addon="script.module.six" version="1.14.0+matrix.2"/>
+  </requires>
+  <extension point="xbmc.python.module" library="lib"/>
+  <extension point="xbmc.addon.metadata">
+    <assets>
+        <icon>icon.png</icon>
+    </assets>
+    <summary lang="en">A fork of the xbmcswift2 project</summary>
+    <description lang="en">A micro framework to enable rapid development of Kodi plugins. TJ Jiang is responsible for modifications in 0.0.9, 0.0.10, 0.0.11 and 0.0.12.</description>
+    <platform>all</platform>
+    <license>GNU General Public License v3 (GPL-3)</license>
```

Comment:
Should be `GPL-3.0-only` or `GPL-3.0-or-later` depending on your interest (SPDX identifiers)